lib/pbench/test/unit/server/query_apis/conftest.py

```diff
@@ -60,3 +60,61 @@ def query_api(
         return response
 
     return query_api
+
+
+@pytest.fixture
+@responses.activate
+def query_api_get(client, server_config, provide_metadata):
+    """
+    Help controller queries that want to interact with a mocked
+    Elasticsearch service.
+
+    This is a fixture which exposes a function of the same name that can be
+    used to set up and validate a mocked Elasticsearch query with a JSON
+    payload and an expected status.
```

Comment:
> which exposes a function of the same name

That's not actually what it does or how it works.  In a test, a reference to a fixture yields the (cached) value which the function of that name returned.  So, "This is a fixture which returns a function that can be...."